docs/semgrep-app/notifications.md

```diff
@@ -112,6 +110,35 @@ If you would like to allow Semgrep to automatically suggest quick fixes, visit t
 
 NOTE: GitLab MR comments are only available to logged-in semgrep.dev users, as they require a Semgrep API token.
 
+### Automatically fix your findings through pull or merge requests
+
+[Autofix](../experiments/overview.md/#autofix) is a Semgrep feature in which rules contain suggested fixes to resolve findings. Either metavariables or regex matches are replaced with a potential fix. Due to their complexity, not all rules make use of autofix, but for rules that use this feature, autofix allows you to quickly resolve findings as part of your code review workflow. Semgrep App can suggest these fixes through PR or MR comments within GitHub or GitLab, thus integrating seamlessly with your review environment.
+
+Autofix is free to use for all tiers.
+
+In the following screenshot, Semgrep detects the use of a native Python XML library, which is vulnerable to XML external entity (XXE) attacks. The PR comment automatically suggests a fix by replacing `import xml` to `import defusedxml`.
+
+
+![Screenshot of a sample autofix PR suggestion](../img/notifications-github-suggestions.png)
+
+
+#### Enabling autofix for your GitLab or GitHub code repository
+
+
+**Prerequisites:**
+
+* **For GitLab users:** GitLab users must be signed in to semgrep.dev as they require a Semgrep API token.
+* **For GitHub users:** Ensure you have granted sufficient permissions requested by Semgrep GitHub App for the repository or organization that will receive PR autofix suggestions.
```

Comment:
IMO these really belong in the PR comments and MR comments sections. The only pre-requisite here is that you have followed the instructions in those sections and are successfully receiving inline PR or MR comments. This makes it sound like there are additional pre-requisites for autofix.